MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -2938,10 +2906,27 @@
       "Id": "ItemGreatBayCompass",
       "RequiredItems": [
         "AreaGreatBayTempleAccess",
-        "Exit Great Bay Temple",
-        "MaskZora"
+        "Exit Great Bay Temple"
+      ],
+      "ConditionalItems": [
+        [
+          "MaskZora"
+        ],
+        [
+          "GBT Cutscene Diving",
+          "GBT Goron Waterwheel",
+          "GBT Red Valve"
+        ],
+        [
+          "GBT Cutscene Diving",
+          "GBT Goron Waterwheel",
+          "FD Jumps"
```

Comment:
Also how does this one work?

Reply:
For the Compass Chest? You can cutscene dive to the Map Chest room, FD jump to the seesaw room, and then dive from the top of the iceless room into the tunnel leading to the Compass Chest room. The dexihands are not hard required to get hurled into the room.